rest/model/user.go

```diff
@@ -91,9 +91,6 @@ func (g *APIGithubUser) BuildFromService(usr user.GithubUser) {
 }
 
 func (g *APIGithubUser) ToService() user.GithubUser {
-	if g == nil {
```

Comment:
Why do we no longer need the nil check here?

Reply:
It's more that we just don't make that check anywhere. Conventionally we don't call ToService on anything uninitialized, and I checked all usages of this function and it's the same way.